sunpy/data/fits_solarnet_schema.yaml

```diff
@@ -0,0 +1,307 @@
+# the following is based on SOLARNET Metadata Recommendations for Solar Observations
+# Version 2.0 – 14. August 2023
+# Stein Vidar Hagfors Haugan and Terje Fredvik
+# https://arxiv.org/pdf/2011.12139.pdf
```

Comment:
I made this comment in the meeting discussion today, but I wonder whether we want to store this schema in `sunpy` itself. As this convention was developed elsewhere, I don't think we want to be the ones that own the spec/become *de facto* in charge of maintaining/updating the spec. My suggestion would be to either pull this schema from its official source or if that doesn't exist in a convenient format, then to put this in a separate, versioned repository that we can pull from such that the schema is maintained separately from `sunpy`.